files/check_openvpn

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Checks if an OpenVPN is avalilable and working.
+
+# Parameters:
+# -c - OpenVPN configuraton that should be used
+# -h - Host name that should be reached through the openvpn
+
+. "$(dirname "${0}")/common.sh"
+checkName=openvpn
+
+# Parameters
+crit=1
+warn=10
+packet_loss=${warn}
+
+usage() {
+	echo "Usage: ${0} -c<config files> -h<host name>"
+	exit 3
+}
+
+while getopts ":h:c:" opt; do
+	case "${opt}" in
+		c)
+			config="${OPTARG}"
+			;;
+		h)
+			host="${OPTARG}"
+			;;
+		\?)
+			echo "Invalid option: -${OPTARG}"
+			usage
+			;;
+		:)
+			echo "Option -${OPTARG} requires an argument"
+			usage
+			;;
+	esac
+done
+
+output="$(timeout -k 1 6 openvpn --config $config || true )"
+output_cleaned="$(echo $output | grep 'Initialization Sequence Completed' || true)"
+#echo $output_cleaned
+if [ -z "${output_cleaned}" ]; then
+	echo "Failed"
+	crit=1
+	warn=1
+else
+	$(timeout -k 1 20 openvpn --config $config &>> /dev/null &)
```

Comment:
Why start another openvpn client here? Just use the old one. Also, quote the string (as always) and you don't need `&>>` because you don't append to an existing file. `&>` is enough